guides/getting-started-esnext.md

```diff
@@ -0,0 +1,189 @@
+# Getting Started with Aurelia
+
+  Aurelia is an amazing framework that embraces simple clean code without sacrificing power.
+  In this tutorial we'll introduce you to the simplicity of Aurelia through construction of a
+  "Todo" application. You'll see just how clean and simple your application code can be and
+  you will learn the many of the basic concepts and capabilities of Aurelia.
+  
+  Before we start writing code. you will need to get setup with the tools we will be using
+  to create the application. We will be using the aurelia comand line tool to create the project
+  structure. The Aurelia CLI will allow to choose between to popular programming language options: ESNext and typescript. In this tutorial we will be using ESnext as our programming language.  
+  
+## Prerequisites
+
+The first thing that you will need to have installed is node.js it can be downloaded 
+from here https://nodejs.org/. Which can be downloaded from the provided link. After following the installation instructions for node. 
+
+The next tool that will need to be installed is yarn, which can
```

Comment:
Although Yarn is a pretty good alternative, I think we should still mention NPM here, and its also faster to start if you don't need to install Yarn.